microsoft-edge/devtools/console/index.md

```diff
@@ -99,7 +99,7 @@ console.table(document.body.getBoundingClientRect());
 let technologies = ["HTML", "CSS", "SVG", "ECMAScript"];
 console.groupCollapsed('Technolgies');
```

Comment:
```suggestion
console.groupCollapsed('Technologies');
```
global: find Technolgies, inspect

Reply:
todo: fix demo's typos in PR 83:

Sync lists of PWA demos #3513 - Docs repo
https://github.com/MicrosoftDocs/edge-developer/pull/3513

Sync lists of PWA demos MicrosoftEdge/Demos#83 - Demos repo
https://github.com/MicrosoftEdge/Demos/pull/83

fixed typos above copied from demo page https://microsoftedge.github.io/Demos/devtools-console/logging-demo.html
* techologies
* Technolgies

@captainbrosset